src/v2/ion/responseTransformer.js

```diff
@@ -66,17 +66,11 @@ const getRemediationValues = (idx) => {
       value: []
     });
   }
-  _.each(idx.neededToProceed, (data) => {
-    if (data && data.name) {
-      // remove stateHandle field
-      if (data.value) {
-        data.value = data.value.filter((formField) => { return !! (formField.visible !== false); } );
-      }
-      remediationValues.push(_.omit(data, 'rel', 'accepts'));
-    }
-  });
   return {
-    remediations: remediationValues
+    remediations: [
+      ...remediationValues,
```

Comment:
just fyi, spread operator is not supported in IE.

Reply:
lucky babel transform it to
```
remediations: [].concat(remediationValues, (0, _toConsumableArray3.default)(idx.neededToProceed))
```